api/proto/dsse.proto

```diff
@@ -0,0 +1,30 @@
+// Copyright 2025 The Sigstore Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package dev.sigstore.rekor.v2;
+
+import "google/api/field_behavior.proto";
+
+import "verifier.proto";
+
+option go_package = "github.com/sigstore/rekor-tiles/pkg/generated/protobuf";
+
+// A request to add a dsse entry to the log
+message DSSERequest {
+    // The full dsse envelope specified as a stringified JSON object
+    string envelope = 1 [(google.api.field_behavior) = REQUIRED];
```

Comment:
It might be useful to add a comment here that you're looking for stringified JSON for reasons of canonicalization OR that the JSON should adhere to https://github.com/secure-systems-lab/dsse/blob/master/envelope.proto

Reply:
done